hom/utils.py

```diff
@@ -119,58 +134,71 @@ async def create_ticket_for_user(interaction, instructions, button_label, exampl
         },
     )
 
-    msg_content = f":envelope:  We have created a support ticket for you, click [here]({new_text_channel.jump_url}) to view."
+    content = (
+        ":envelope:  We have created a support ticket for you, click [here]"
+        f"({new_text_channel.jump_url}) to view."
+    )
 
     embed = discord.Embed(description=instructions, title=button_label)
-    embed.set_image(url=example_url) if example_url else None
-
     embed.set_footer(
-        text="This channel is only visible to you and our moderators. If your question has been answered, feel free to close the ticket."
+        text=(
+            "This channel is only visible to you and our moderators. If your question has been "
+            "answered, feel free to close the ticket."
+        )
     )
-    await interaction.followup.send(content=msg_content, ephemeral=True)
+
+    if example_url:
+        embed.set_image(url=example_url)
+
+    await interaction.followup.send(content, ephemeral=True)
     await new_text_channel.send(
-        content=f"{interaction.user.mention}", embed=embed, view=vw.PViewSupport_Message()
+        f"{interaction.user.mention}", embed=embed, view=views.SupportMessage()
     )
 
-    await send_log_message(
-        interaction=interaction,
-        content=f"({new_text_channel.topic}) Ticket opened for user:\n``{interaction.user.display_name}`` - {interaction.user.mention}",
-        mod=interaction.client.user,
+    log_content = (
+        f"({new_text_channel.topic}) Ticket opened for user:\n``{interaction.user.display_name}`` "
+        f"- {interaction.user.mention}"
     )
 
+    assert interaction.client.user
+    await send_log_message(interaction, log_content, interaction.client.user)
     return new_text_channel
 
 
-async def send_log_message(interaction, content, mod, channel=None):
+async def send_log_message(
+    interaction: discord.Interaction,
+    content: str,
+    mod: discord.User | discord.ClientUser | discord.Member,
+    channel: t.Optional[discord.guild.GuildChannel] = None,
+) -> t.Optional[discord.Message]:
+    assert interaction.guild
```

Comment:
Can an interaction ever not have a guild? (again, still unfamiliar with assert statements)